src/base/bittorrent/bencoderesumedatastorage.cpp

```diff
@@ -409,6 +426,13 @@ void BitTorrent::BencodeResumeDataStorage::Worker::store(const TorrentID &id, co
     data["qBt-firstLastPiecePriority"] = resumeData.firstLastPiecePriority;
     data["qBt-stopCondition"] = Utils::String::fromEnum(resumeData.stopCondition).toStdString();
 
+    if (const QSslCertificate &cert = resumeData.sslParameters.certificate; !cert.isNull())
+        data[KEY_SSL_CERTIFICATE] = cert.toPem().toStdString();
+    if (const QSslKey &privateKey = resumeData.sslParameters.privateKey; !privateKey.isNull())
+        data[KEY_SSL_PRIVATE_KEY] = privateKey.toPem().toStdString();
+    if (const QByteArray &dhParams = resumeData.sslParameters.dhParams; !dhParams.isEmpty())
+        data[KEY_SSL_DH_PARAMS] = dhParams.toStdString();
```

Comment:
I don't see much point in declaring new names for these fields. The following doesn't look any less readable, IMO:
```suggestion
    if (!resumeData.sslParameters.certificate.isNull())
        data[KEY_SSL_CERTIFICATE] = resumeData.sslParameters.certificate.toPem().toStdString();
    if (!resumeData.sslParameters.privateKey.isNull())
        data[KEY_SSL_PRIVATE_KEY] = resumeData.sslParameters.privateKey.toPem().toStdString();
    if (!resumeData.sslParameters.dhParams.isEmpty())
        data[KEY_SSL_DH_PARAMS] = resumeData.sslParameters.dhParams.toStdString();
```

Reply:
All comments addressed. PR updated.